src/pyproject.toml

```diff
@@ -0,0 +1,14 @@
+[tool.black]
+line-length = 88
```

Comment:
can we do 80-81? I admit it's mostly for my benefit, because

![image](https://user-images.githubusercontent.com/506918/118462271-05295a80-b6ee-11eb-8b23-533efb80f9a7.png)

Reply:
okay, in retrospect, I think I can live with the line wrapping. Mostly because I think `black` will do the right thing when it comes to it:

![image](https://user-images.githubusercontent.com/506918/118463621-7fa6aa00-b6ef-11eb-855c-949fdaf562cb.png)

Reply:
While I'm fine with 81, you seem to have plenty of blank space in those files?

Reply:
because those lines are not close to 80 cols. ;)

Reply:
Can we, please? If no one particularly opposes to it, that is. I can live with the 88 columns, but it's just easier to  have the 3 pane splits which wrap at 80 each, instead of having to get rid of one because of annoying line wrapping. :)